src/main/webapp/app/components/oncokbTable/HeaderConstants.tsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { ToggleValue } from 'app/pages/usageAnalysisPage/UsageAnalysisPage';
+
+export const emailHeader = <span>Email</span>;
+export const usageHeader = <span>Usage</span>;
+export const resourceHeader = <span>Resource</span>;
+export const endpointHeader = <span>Most frequently used endpoint</span>;
+export const noPrivateEndpointHeader = (
+  <span>Most frequently used endpoint(only public)</span>
+);
+export const timeHeader = <span>Time</span>;
+export const operationHeader = <span>Details</span>;
```

Comment:
- We should at minimum Uppercase the first letter since they are components.
- I would not prefer wrap constant string in a span component. It's just unnecessary <span> when you wrap it in other tags
- The table header could be string